### ISSUE TITLE
Set epoch using --n_epoch

### ISSUE THREAD

**Issue by darrenleeleelee1** [junyanz/pytorch-CycleGAN-and-pix2pix#1185]
I am use argument --n_epoch but  it seems not work, is --n_epoch is setting max epoch to what I want or is other command?

**Comment by JunlinHan** [comment]
Epoch has 2 parts, see ./options/train_option.
parser.add_argument('--n_epochs', type=int, default=100, help='number of epochs with the initial learning rate')
parser.add_argument('--n_epochs_decay', type=int, default=100, help='number of epochs to linearly decay learning rate to zero')
So if you want to train max to 400 epoch and you want to keep the learning rate unchanged before 300. You can set n_epochs=300 and n_epochs_decay = 100.
